### ISSUE TITLE
[feature request] hourly forecast for today

### ISSUE THREAD

**Issue by ryannathans** [amit9838/mousam#166]
There is hourly history for today
There is hourly forecast for tomorrow
How about hourly forecast for today? I'd like to see what temperature it will be in a few hours!

**Comment by ryannathans** [comment]
![image](https://github.com/user-attachments/assets/665f6a39-22d4-463a-9e99-dfd6a553345b)
Figured I would attach a screenshot incase the "hourly history" is supposed to be an hourly forecast

**Comment by amit9838** [comment]
can you also share , when it starts? I mean the start time.

**Comment by ryannathans** [comment]
> can you also share , when it starts? I mean the start time.
right
![image](https://github.com/user-attachments/assets/62374646-2d64-4b2d-805f-9c4ec078dbb7)
left
![image](https://github.com/user-attachments/assets/cb99486a-44b5-4200-8299-b190d018a2f7)

**Comment by ryannathans** [comment]
![image](https://github.com/user-attachments/assets/be867907-9398-43fa-9fa1-5bad610b53d9)


Today it seems to be working properly, no idea what changed

**Comment by travisfw** [comment]
Yes my hourly forecast for today always ends at 3PM. Surely there is more data available.

![Image](https://github.com/user-attachments/assets/ffb330c6-fd02-4c98-86d9-94fbb4f4b9c8)

**Comment by roshanshariff** [comment]
Looking at the [openMeteo docs](https://open-meteo.com/en/docs), it looks like you need to set the `timezone` parameter in the API call. Without that, it assumes GMT and the hourly data ends at midnight GMT. It probably makes more sense to set `timezone=auto` to get data for the entire day in the timezone of the forecast location. The other alternative would be to use the local time zone of the user. I believe this is a relatively small change in `src/backendWeather.py`.
